### DISCUSSION TITLE
Incorrect "following" count in my profile

### DISCUSSION THREAD

**Discussion by 1vanK** [community/community#73299] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

https://github.com/1vanK

The profile currently says "51 followers · 0 following", but if you click on the link "0 following" a list will be showed

**Comment by Dineth-H** [comment]
A bug. It's possible that there is a bug in the platform that is causing this discrepancy. If you think this is the case, you can report the bug to the [platform's support team](https://support.github.com/contact/account). 

A glitch. It's also possible that there is a temporary glitch in the system. If this is the case, it should resolve itself on its own within a short period of time.

**Comment by IRNCyber** [comment]
### **Bug Report**: Inconsistent Follower/Following Counts on Profiles  

#### **Description**  
There seems to be a mismatch between the follower and following counts displayed on profiles and the actual lists:  

- On [this profile](https://github.com/1vanK), the header shows **"51 followers · 0 following"**, but clicking on "0 following" reveals a list, suggesting the count is incorrect.  
- On my profile, [freedom-foundation](https://github.com/freedom-foundation?tab=followers), the header displays **4 followers**, but only **3 followers** are visible in the list.  

#### **Steps to Reproduce**  
1. Visit any GitHub profile where this inconsistency might occur.  
2. Compare the displayed follower/following count with the actual number in the respective list.  
3. Note any discrepancies.  

#### **Expected Behavior**  
The counts shown on the profile header should perfectly match the entries in the followers and following lists.  

#### **Observed Behavior**  
The numbers in the header do not align with the actual lists, leading to confusion.  

#### **Why It Matters**  
✨ **Clarity & Trust**: Accurate metrics build trust among users.  
✨ **Profile Integrity**: Misleading counts might cause confusion about engagement levels.  

#### **Request**  
Please investigate and resolve this inconsistency to ensure profiles reflect accurate data. Your attention to this issue is greatly appreciated! 🙌✨
